cmd/kusk/.goreleaser.yml

```diff
@@ -0,0 +1,41 @@
+before:
```

Comment:
Can you explain why `.goreleaser.yml` is almost the same as this? Basically, is this needed?

Reply:
Even though Kusk is now in the same repo, i wanted them to be seperate entities. Having seperate files keeps it readable and we can test the release in isolation

Reply:
makes sense